chsdi/locale/empty_chsdi.po

```diff
@@ -8860,3 +8860,33 @@ msgstr ""
 
 msgid "klimaeignung_zone"
 msgstr ""
+
+msgid "tt_eta_ch"
```

Comment:
Can you answer my question please? Can you search the features in the search field or in the query tool? If you don't know what it is just ask me.